pyproject.toml

```diff
@@ -14,12 +14,12 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.6.1,<4"
-django = ">=2.2"
+python = ">=3.8,<4"
+django = ">=3.2"
 daemonize = "~=2.5.0"
 prometheus-client = "~=0.7"
 typing-extensions = "^4"
-redis = {version = "~=3.*", optional = true}
+redis = {version = ">=3.0,<5", optional = true}
```

Comment:
Any reason not to just go with `^4`? `redis<4` seems to be old enough to include Python 2.7 support. Given that this introduces a  Python version 'breaking change' anyway I'd have thought that this would at least be a minor version bump and so requiring a semi-modern `redis` version feels reasonable especially given that `redis`'s only requirement is `async-timeout` (which has no additional requirements`).